kernelci/cli/base.py

```diff
@@ -178,6 +178,11 @@ class Args:  # pylint: disable=too-few-public-methods
         'help': "Path to the dtbs.json file",
     }
 
+    group_id = {
+        'name': 'group_id',
+        'help': "User group id",
+    }
+
```

Comment:
I'm not sure it's useful to have it in the shared class, but that's a grey area anyway.  All the other arguments there are optional ones with `--something`.  This is not a blocking issue but rather something that would need to be revisited once we remove all the deprecated command line tools (or move the code to the `kernelci.legacy` package).